### ISSUE TITLE
Verständnisfrage zur Bewertung

### ISSUE THREAD

**Issue by Andreas-Englisch** [BIK-BITV/BIK-Web-Test#459]
Im BIK BITV-Test können einzelne Kriterien nur als "erfüllt" und "nicht erfüllt" bewertet werden, andere hingegen zusätzlich mit 3 weiteren Abstufungen (https://bitvtest.de/test-methodik/web/beschreibung-des-pruefverfahrens)

Woran erkenne ich bei den Prüfschritten, wie sie bewertet werden können?

Beispiel: 
* [9.1.3.2](https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-3-1-2-anderssprachige-woerter-und-abschnitte-ausgezeichnet): Hier wird nur "erfüllt" und "nicht erfüllt" genannt, aber die Beschreibung der Bewertung ("weniger als 50%", "zu mindestens 90%") deutet daraufhin, dass auch die Zwischenstufen erlaubt sind
* [9.1.3.1](https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-3-1-1-hauptsprache-angegeben): Auch hier wird nur "erfüllt" und "nicht erfüllt" genannt und da die Sprache nur korrekt ausgezeichnet sein kann oder eben nicht, scheint es sinnvoll, dass es nur die zwei Bewertungsalternativen gibt (obwohl selbst das nur auf den ersten Blick so offensichtlich klar zu sein scheint, denn wäre ein "en-US" bei einer australischen Seite ("en-AU") besser nur "eher erfüllt" statt "erfüllt" oder "nicht erfüllt?)
*  [9.2.5.3](https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-2-5-3-sichtbare-beschriftung-teil-des-zugaenglichen-namens): Hier wird nur "erfüllt" genannt und mir ist unklar, ob alle Verstöße dann "nicht erfüllt" sind oder abgestuft bewertet werden dürfen
* [9.2.4.7](https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-2-4-7-aktuelle-position-des-fokus-deutlich): Hier wird neben "erfüllt" und "nicht erfüllt" auch das m.E. nirgendwo definierte "nicht voll erfüllt" genannt und mir ist unklar, was damit gemeint ist: alles zwischen "eher erfüllt" und "eher nicht erfüllt"?

**Comment by Andreas-Englisch** [comment]
Hallo Johannes, Danke für Deine Antwort! 

In der Beschreibung des Prüfverfahrens steht:

> Für einige wenige Prüfschritte sind nur die beiden Bewertungsalternativen "erfüllt" und "nicht erfüllt" vorgesehen

Ich bin davon ausgegangen, dass diese Prüfschritte klar definiert sind. Vielleicht wurde nur vergessen, dies zu dokumentieren?

**Comment by johannesFischer84** [comment]
Ja, stimmt, zumindest in der Vergangenheit gab es diesen Fall, beispielsweise bei WCAG-Kriterium 4.1.1. Je mehr ich dieses Kriterium geprüft habe, umso eher war ich aber auch hier der Meinung, dass "eher erfüllt" auch möglich sein sollte.
Die klare Definition, welche Stufen man vergeben "darf", ist mit den Jahren in den Prüfschrittbeschreibungen eher herausgefallen, glaube ich.

"erfüllt" und "nicht erfüllt" entspricht ja quasi den WCAG-Bewertungen "konform" und "nicht konform", und mehr gibt es in den WCAG auch nicht. Daran versuche ich mich im Grunde in allen Prüfungen zu orientieren. Aber wenn man noch etwas durchgehen lassen kann, ist es dann eben "eher erfüllt". Ob man dann noch "teilweise erfüllt", "eher nicht erfüllt" und "nicht erfüllt" abstuft, bringt aus meiner Sicht in den meisten Fällen wenig Mehrwert.
